### PR TITLE
Updated for Vizzu 0.4.6

### PR DESCRIPTION
**Pull request by simzer**
From version Vizzu v0.4.6 there is no need to manually copy the wasm file to the public folder since its location can be set for Vizzu.

### CODE REVIEW

File: src/pages/index.tsx [hunk 1]
```diff
@@ -1,5 +1,6 @@
 import React, { FC, useEffect, useRef } from 'react'
 import Vizzu, { AnimTarget } from 'vizzu'
+import VizzuModule from './../../node_modules/vizzu/dist/cvizzu.wasm?url'
```
Comment (christopher-caldwell) [inline src/pages/index.tsx]: I cloned this, and changed this path to `import VizzuModule from 'vizzu/dist/cvizzu.wasm?url'`. Looks like the relative pathing isn't needed here.

**Comment by christopher-caldwell** [comment]
Looks good to me. Pull the changes and it still works. Only thing I'd suggest changing is the relative pathing.

**Comment by simzer** [comment]
Thanks, I commited the suggested change.

**Comment by simzer** [comment]
And added a small description about how to add Vizzu to an existing project into the readme.

**Comment by christopher-caldwell** [comment]
> And added a small description about how to add Vizzu to an existing project into the readme.

Would you be able to change this to yarn? or even add a yarn and npm example? The bottom half of the readme, and the template itself use yarn. 

Further, is there a reason to use `let` instead of `const` for the chart declaration?

**Comment by simzer** [comment]
Thanks, I have fixed them.

**Pull request merged**